### ISSUE TITLE
Cannot Load *certain* hi-res textures

### ISSUE THREAD

**Issue by richard42** [mupen64plus/mupen64plus-user-issues#433]
Originally reported on Google Code with ID 434

```
Describe your system:
 - Operating System (be specific): Ubuntu 11.04 - Unity Removed
 - Machine type (32-bit or 64-bit): 64-Bit
 - Mupen64Plus version: 1.99.4
 - Plugins used: Rice 1.99.4

quack@Quack-Igor-Ubuntu:~$ mupen64plus --set Video-Rice[LoadHiResTextures]=True --saveoptions
~/.ROMS/Super\ Mario\ 64\ \(U\)\ \[\!\].z64  __  __                         __   _
 _   ____  _             
|  \/  |_   _ _ __   ___ _ __  / /_ | || | |  _ \| |_   _ ___ 
| |\/| | | | | '_ \ / _ \ '_ \| '_ \| || |_| |_) | | | | / __|  
| |  | | |_| | |_) |  __/ | | | (_) |__   _|  __/| | |_| \__ \  
|_|  |_|\__,_| .__/ \___|_| |_|\___/   |_| |_|   |_|\__,_|___/  
             |_|         http://code.google.com/p/mupen64plus/  
Mupen64Plus Console User-Interface Version 1.99.4

UI-console: attached to core library 'Mupen64Plus Core' version 1.99.4
            Includes support for Dynamic Recompiler.
Core: Goodname: Super Mario 64 (U) [!]
Core: Name: SUPER MARIO 64
Core: MD5: 20B854B239203BAF6C961B850A4A51A2
Core: CRC: 635a2bff 8b022326
Core: Imagetype: .z64 (native)
Core: Rom size: 8388608 bytes (or 8 Mb or 64 Megabits)
Core: Version: 1444
Core: Manufacturer: Nintendo
Core: Country: USA
UI-Console: Cheat codes disabled.
UI-console: using Video plugin: 'Mupen64Plus OpenGL Video Plugin by Rice' v1.99.4
UI-console: using Audio plugin: 'Mupen64Plus SDL Audio Plugin' v1.99.4
Input: N64 Controller #1: Using SDL joystick 0 ('Microsoft X-Box 360 pad')
UI-console: using Input plugin: 'Mupen64Plus SDL Input Plugin' v1.99.4
UI-console: using RSP plugin: 'Hacktarux/Azimer High-Level Emulation RSP Plugin' v1.99.4
Input: N64 Controller #1: Using SDL joystick 0 ('Microsoft X-Box 360 pad')
Input: 1 controller(s) found, 1 plugged in and usable in the emulator
Input: Rumble activated on N64 joystick #1
Input: Mupen64Plus SDL Input Plugin version 1.99.4 initialized.
Video: SSE processing enabled.
Video: Found ROM 'SUPER MARIO 64', CRC ff2b5a632623028b-45
Video: Texture loading option is enabled. Finding all hires textures
Video: Initializing OpenGL Device Context.
Core: Setting 32-bit video mode: 640x480
Video Warning: Failed to set GL_DEPTH_SIZE to 16. (it's 24)
Video: Using OpenGL: NVIDIA Corporation - GeForce GTX 480/PCI/SSE2 : 4.1.0 NVIDIA 270.41.06
Video: OpenGL Combiner: Fragment Program
Audio: Initializing SDL audio subsystem...
/dev/mixer: : No such file or directory
Core: Starting R4300 emulator: Dynamic Recompiler
Core: R4300: starting 64-bit dynamic recompiler at: 0x7f4974220200
/dev/mixer: : No such file or directory

Describe the problem:

Certain Textures will not load when using Rice 1.99.4.

Have also created a thread at Emutalk:

http://www.emutalk.net/threads/53086-Mupen64plus-Unable-to-load-*CERTAIN*-Hi-res-textures.?p=440912#post440912

```

Reported by `james.walker117` on 2011-05-30 11:52:40

**Comment by richard42** [comment]
```
Several problems in the Rice texture loading code which cause this symptom have been
fixed since the 1.99.4 release.  Please re-test with a new build from the latest code
and report your results.
```

Reported by `richard42g` on 2011-06-01 03:47:51
- **Status changed**: `Accepted`
- **Labels added**: Component-RiceVideo, Type-Defect

**Comment by richard42** [comment]
```
I assume this defect was fixed; reporter has not tested in last 9 months, so closing.
```

Reported by `richard42g` on 2012-02-25 15:23:06
- **Status changed**: `Fixed`
